charts/pega/values.yaml

```diff
@@ -15,8 +15,8 @@ global:
     execute: "deploy"
 
   # Add custom certificates to be mounted to container
-  #to support external secrets
-  #supports multiple externalSecrets , add externalSecret names as array of strings
+  # to support external secrets
```

Comment:
to support an external secret, specify your secret using the certificates parameter.
to support multiple external secrets, specify each of your external secrets as an array of comma-separated strings using the certificatesSecrets parameter.